examples/basic_usage.jl

```diff
@@ -40,99 +42,109 @@ end
 # Load environment variables
 env_loaded = load_env_file(joinpath(@__DIR__, ".env"))
 
-# Test library loading
-println("Using default library path")
-load_iceberg_library()
-println("✅ Library loaded!")
+# Initialize Iceberg runtime
```

Comment:
Would you mind switching this to a real Julia test (either with ReTestItems or using Test)?

Reply:
The only thing is that it should still be possible to run it separately of other tests, and with custom `ICEBERG_RUST_LIB`

Reply:
I mean we should have an integration test that exercises the usage. We can then remove the usage file